scripts/core/spellcheck/spellcheck.ts

```diff
@@ -8,7 +8,7 @@ import {debounce, once} from 'lodash';
 SpellcheckService.$inject = ['$q', 'api', 'dictionaries', '$rootScope', '$location', 'lodash', 'preferencesService'];
 function SpellcheckService($q, api, dictionaries, $rootScope, $location, _, preferencesService) {
     var PREFERENCES_KEY = 'spellchecker:status',
-        lang,
+        lang = 'en',
```

Comment:
Would abbreviations work in another language with this code? I think it'd better to make sure that a correct language is being set without hard-coding values